lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
@twiest Between this and the latter comment regarding returning None for non-existent keys, I think that is a good approach.

So, removing the idea of having a default or even needing to use the boolean param for the default filter, will provide consistent behavior across the board, so if a variable is set on the command line it will override anything set in the env and either will override the default provided by the default filter.